src/lib/mixer_module/mixer_module.cpp

```diff
@@ -115,6 +131,53 @@ void MixingOutput::updateParams()
 		_mixers->set_thrust_factor(_param_thr_mdl_fac.get());
 		_mixers->set_airmode((Mixer::Airmode)_param_mc_airmode.get());
 	}
+
+	/** Update Function Mappings */
+
+	updateParamValues("FUNC", _assigned_function);
+	updateFailsafeValues();
+	updateDisarmedValues();
+	updateMinValues();
+	updateMaxValues();
+	updateTrimValues();
+	updateReverseMask();
+
+	// Determine the output_control groups required to subscribe to
+
+	// First, clear all bits associated with the output_control groups
+	_groups_required &= (1 << n_act) - 1;
```

Comment:
The logic around required/subscribed groups is complex. Is there a big performance impact vs always subscribing?

Reply:
Well, with so many topics, you don't necessarily want every output module responding to every single topic subscription; especially if you have, say, one module controlling a gimbal + some other one-off servos, and a few other modules controlling the motors and flight control surfaces.  They don't need to all update and run on every publication of every topic.  I agree it does seem a bit messy, though.